src/Packaging.NServiceBus.AzureServiceBus/Packaging.NServiceBus.AzureServiceBus.csproj

```diff
@@ -39,22 +38,22 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
-    <Reference Include="System.Runtime.Serialization" />
-    <Reference Include="System.ServiceModel" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
     <Reference Include="System.Xml" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\ServiceControl.Transports.AzureServiceBus\ServiceControl.Transports.AzureServiceBus.csproj">
-      <Project>{66ca123a-7e48-494f-a3d3-ded5dd664621}</Project>
+      <Project>{a5a18ce5-2065-4e9a-a8a3-e55cabb50407}</Project>
       <Name>ServiceControl.Transports.AzureServiceBus</Name>
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Target Name="AfterBuild">
```

Comment:
Why was this removed, but added in the new Packaging.NServiceBus.AzureServiceBusStandard.csproj file?